src/main/java/lotto/domain/Lotto.java

```diff
@@ -0,0 +1,59 @@
+package lotto.domain;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Objects;
+
+public class Lotto {
+    private final List<LottoNumber> numbers;
+
+    public Lotto(String[] numbers) {
+        this(StringToList(numbers));
+    }
+
+    public Lotto(List<LottoNumber> numbers) {
+        this.numbers = numbers;
+    }
+
+    private static List<LottoNumber> StringToList(String[] numbers) {
+        List<LottoNumber> list = new ArrayList<>();
+        for (int i = 0; i < numbers.length; i++) {
+            list.add(new LottoNumber(Integer.parseInt(numbers[i])));
+        }
+        return list;
+    }
+
+    public int matchedCount(Lotto lotto) {
+        int cnt = 0;
+
+        for (int i = 0; i < lotto.value().size(); i++) {
```

Comment:
```suggestion
        for (LottoNumber number: lotto.value()) {
```
iter 구문으로 구현하는 습관 만들어 본다.